### ISSUE TITLE
Custom Prompt Names

### ISSUE THREAD

**Issue by devfag** [cloveranon/Clover-Edition#65]
This one would require assistance from the pastebin anons, but I think it's worth it.

Basically, the idea is that instead of adding names to prompts, you substitute them with `(you)` and `(them1)`, `(them2)`, `(them3)`, etc.

When you load a custom prompt from a file, it displays the prompt to the user, and allows them to enter in a name for yourself, and a unique name for each unique `(them#)` in the prompt. It then substitutes them.

**Comment by OpenCYOAI** [comment]
Also, concerning the prompts, if we start using the colab's wiki for them, maybe scrape the prompts from there? since markdown is basically a html wrapper, it could be easier to grab the prompts from there instead of pastebin. I could also add one of you as a colaborator on the colab's repo to allow you to manage the tags of the prompts, to make it easier to manage the scraper

**Comment by devfag** [comment]
Yeah, I think scraping the wiki is a good idea. I can help format the prompts and set up the naming conventions if you want.

**Comment by OpenCYOAI** [comment]
It's mainly for you, you're the ones scraping to get the custom prompts. The idea of the proposal is to allows you to have cross-access to your scraper on a side, and the prompts and their tags on the other. If you manage to organize them in a way they are still easily readible on the wiki, while in the same time, organize their tags to easy the development of your scraper, it's win-win. sending you an invite soon

**Comment by cloveranon** [comment]
Would not recommend linking to anything on github, it is unlikely to last very long.
